docs/getting-started.md

```diff
@@ -109,10 +109,10 @@ var app = shunter({
 app.start();
 ```
 
-When we run this file using `node app.js`, you should see a bunch of logs ending with a message like this:
+When we run this file using `node app.js`, you should see a short delay then a message like this:
 
 ```
-All child processes listening
+Shunter started with 4 child processes listening
```

Comment:
Yes, or the `-c` option if provided.  I intended people to read the "a message like this" to mean similar to.  Do you think it might be clearer to use "X" instead of 4?